# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/chrome.toml
# Commit: e47abb2b8ac7
# Full Hash: e47abb2b8ac70efe89371c67766def82374c0f36
# Author: Alexandru Marc <amarc@mozilla.com>
# Date: 2025-09-27 12:29 +0300
# Description:
#   Revert "Bug 1968890 - Part 9: Add test for dynamic import. r=jonco" as potential cause of javascript engine crashes (bug 1991268)
# ==============================================================================

diff -r 7e2611c97326 -r e47abb2b8ac7 dom/base/test/jsmodules/chrome.toml
--- a/dom/base/test/jsmodules/chrome.toml	Sat Sep 27 11:41:54 2025 +0300
+++ b/dom/base/test/jsmodules/chrome.toml	Sat Sep 27 12:29:56 2025 +0300
@@ -1,7 +1,6 @@
 [DEFAULT]
 support-files = [
   "ambiguous_export.mjs",
-  "import_404.mjs",
   "import_ambiguous.mjs",
   "import_ambiguous_indirect_export.mjs",
   "import_ambiguous_export.mjs",